extension/src/prototypes/view/selection/LayerContent.tsx

```diff
@@ -156,6 +158,12 @@ export function LayerContent<T extends SupportedLayerType>({
 
   const layerName = layerTypeNames[type] ?? rootLayer?.layerName;
 
+  const areas = useAtomValue(areasAtom);
+
+  if (areas == null) {
+    return null;
+  }
```

Comment:
You don't need to use `areasAtom`. You can refer [rawDataset](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/183ef179a8738f1ca683701f6bc2b2bafb7c935c/extension/src/shared/view-layers/rootLayer.ts#L85) from [rootLayerConfig](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/183ef179a8738f1ca683701f6bc2b2bafb7c935c/extension/src/prototypes/view/selection/LayerContent.tsx#L62).